api/tests/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(api-tests)
+
```

Comment:
Is it possible for CMake to run the `initialize_vcan` script instead of requiring the user to do it? It's easy to forget and then you see failed tests and have momentary panic.

Reply:
881472f